packages/viewer/src/commands/streamCommands.ts

```diff
@@ -50,6 +50,20 @@ export function startStream(
   };
 }
 
+export function reconnect(
+  streamId: UUID.UUID,
+  dimensions: Dimensions.Dimensions
+): Command<Promise<Disposable>> {
```

Comment:
Looks like this should return a `vertexvis.protobuf.stream.IStreamResponse` instead.